ulib/axstd/src/os.rs

```diff
@@ -3,4 +3,6 @@
 /// ArceOS-specific definitions.
 pub mod arceos {
     pub use arceos_api as api;
+    #[doc(no_inline)]
+    pub use arceos_api::modules;
```

Comment:
Previously, ning did not expose these modules. What was ning's consideration for exposing all of them in the API now? Is there any safety concern if methods or vars in those modules is not exported as `pub(crate)`

Reply:
To use `axhal` in apps without adding it to dependencies.